libgalois/src/PropertyGraph.cpp

```diff
@@ -52,8 +52,8 @@ MapTopology(const tsuba::FileView& file_view) {
     return katana::ErrorCode::InvalidArgument;
   }
 
-  uint64_t num_nodes = data[2];
-  uint64_t num_edges = data[3];
+  const uint64_t num_nodes = data[2];
+  const uint64_t num_edges = data[3];
```

Comment:
Does this matter. Why?

Reply:
Defensive programming. I rely on these values for all conversions below so don't want anything to change them.

Reply:
More of this please. :-) I approve!